src/pages/instances/InstanceList.tsx

```diff
@@ -535,30 +521,42 @@ const InstanceList: FC = () => {
                       "u-hide": panelParams.instance,
                     })}
                   />
-                  <SelectableMainTable
-                    headers={getHeaders(userHidden.concat(sizeHidden))}
-                    rows={pagination.pageData}
-                    sortable
-                    className="instance-table"
-                    id="instance-table-wrapper"
-                    emptyStateMsg={
-                      isLoading ? (
-                        <Loader text="Loading instances..." />
-                      ) : (
-                        <>No instance found matching this search</>
-                      )
-                    }
-                    itemName="instance"
-                    parentName="project"
-                    selectedNames={selectedNames}
-                    setSelectedNames={setSelectedNames}
-                    processingNames={processingNames}
-                    totalCount={instances.length}
-                    filteredNames={filteredInstances.map(
-                      (instance) => instance.name
-                    )}
-                    onUpdateSort={pagination.updateSort}
-                  />
+                  <ScrollableTable
+                    dependencies={[
+                      instances,
+                      creationOperations,
+                      notify.notification,
+                      filters.queries,
+                      filters.statuses,
+                      filters.types,
+                      filters.profileQueries,
+                      pagination.pageSize,
+                      pagination.currentPage,
+                    ]}
```

Comment:
This looks quite chaotic. Can we create a `const` for this array up there and reference it here?

Reply:
I think I found a simplification to pass the notification and the filtered object list. The rest is not needed.